### PR TITLE
Improve terminal's Table

### PR DESCRIPTION
**Pull request by slack-coder**
Add a little documentation and make it easier to use by accepting `&str` over `String`.

### CODE REVIEW

**Review by FintanH — COMMENTED** [review]
I'm not sure the change to `&str` makes sense because you're allocating to a `String` anyway. But maybe it could take an array of `impl ToString`?

**Comment by slack-coder** [comment]
> I'm not sure the change to `&str` makes sense because you're allocating to a `String` anyway. But maybe it could take an array of `impl ToString`?

Performance is low priority for the Table, but its now less verbose and easier to use.  At least that was the goal.

File: radicle-cli/src/commands/ls.rs [hunk 1]
```diff
@@ -53,10 +53,10 @@ pub fn run(_options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
...
         table.push([
-            term::format::bold(name),
-            term::format::tertiary(id),
-            term::format::secondary(head),
-            term::format::italic(description),
+            &term::format::bold(name),
+            &term::format::tertiary(id),
+            &term::format::secondary(head),
+            &term::format::italic(description),
         ]);
```
Comment (FintanH) [inline radicle-cli/src/commands/ls.rs:60]: I don't think the `&` are needed anymore
```suggestion
            term::format::bold(name),
            term::format::tertiary(id),
            term::format::secondary(head),
            term::format::italic(description),
        ]);
```

File: radicle-cli/src/terminal/patch.rs [hunk 2]
```diff
@@ -56,8 +56,8 @@ pub fn list_commits(commits: &[git::raw::Commit]) -> anyhow::Result<()> {
             .summary_bytes()
             .unwrap_or_else(|| commit.message_bytes());
         table.push([
-            term::format::secondary(term::format::oid(commit.id())),
-            term::format::italic(String::from_utf8_lossy(message)),
+            &term::format::secondary(term::format::oid(commit.id())),
+            &term::format::italic(String::from_utf8_lossy(message)),
```
Comment (FintanH) [inline radicle-cli/src/terminal/patch.rs:60]: Same here :)

**Review by FintanH — APPROVED** [review]
Nice :+1:

**Review by FintanH — APPROVED** [review]

**Pull request merged**